spanner/api/Spanner.Samples.Tests/UpdateUsingDmlWithStructCoreAsyncTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class UpdateUsingDmlWithStructCoreAsyncTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public UpdateUsingDmlWithStructCoreAsyncTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestUpdateUsingDmlWithStructCoreAsync()
+    {
+        UpdateUsingDmlWithStructCoreAsyncSample sample = new UpdateUsingDmlWithStructCoreAsyncSample();
+        var rowCount = await sample.UpdateUsingDmlWithStructCoreAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId);
+        Assert.True(rowCount >= 0);
```

Comment:
Based on samples and test this will be zero as record with first name as `Timothy` and `Campbell` as last name has been added into `WriteAndReadUsingDmlCoreAsync` sample and test for same will be run after this sample's test.